docs/dapp/sapphire/README.mdx

```diff
@@ -42,7 +46,20 @@ fine-grained confidentiality, check out the
 - Chain ID:
   - Hex: `0x5aff`
   - Decimal: `23295`
+- Testing token [Faucet][faucet]
+
+[faucet]: https://faucet.testnet.oasis.io/
+
+### Localnet
+
+- Network name: `sapphire-localnet`
+- Long network name: `Localnet`
+- Chain ID:
+  - Hex: `0x5afd`
+  - Decimal: `23293`
+- Local development Docker [image][localnet]
 
+[localnet]: https://github.com/oasisprotocol/oasis-web3-gateway/tree/main/docker
```

Comment:
While you're at it, can you also change the order of ParaTimes in the readme file here:  https://github.com/oasisprotocol/oasis-web3-gateway/tree/main/docker

so that Sapphire is first and Emerald is second.

Reply:
https://github.com/oasisprotocol/oasis-web3-gateway/pull/620